### PR TITLE
[LoongArch] Support .option directive

### PR DESCRIPTION
**Pull request by wangleiat**
The .option can accept 4 parameters like the LoongArch's gnu as:
push, pop, relax and norelax.

### CODE REVIEW

File: llvm/test/MC/LoongArch/Directives/option-invalid.s [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# RUN: not llvm-mc --triple loongarch64 < %s 2>&1 \
```
Comment (SixWeining) [inline llvm/test/MC/LoongArch/Directives/option-invalid.s:1]: --triple=loongarch64

Comment (SixWeining) [inline llvm/test/MC/LoongArch/Directives/option-invalid.s:1]: '`<`' is useless.

File: llvm/test/MC/LoongArch/Directives/option-pushpop.s [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# RUN: llvm-mc --triple loongarch64 --mattr=-relax < %s \
```
Comment (SixWeining) [inline llvm/test/MC/LoongArch/Directives/option-pushpop.s:1]: Ditto.

File: llvm/test/MC/LoongArch/Directives/option-pushpop.s [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+# RUN:     | FileCheck --check-prefix=CHECK-ASM %s
+# RUN: llvm-mc --triple loongarch64 --filetype=obj < %s \
+# RUN:     | llvm-readobj -r - | FileCheck --check-prefix=CHECK-RELOC %s
+
+# Test the operation of the push and pop assembler directives when
```
Comment (SixWeining) [inline llvm/test/MC/LoongArch/Directives/option-pushpop.s:6]: Use two `#`s as non-check comments.

File: llvm/test/MC/LoongArch/Directives/option-pushpop.s [hunk 4]
```diff
@@ -0,0 +1,59 @@
+# RUN: llvm-mc --triple loongarch64 --mattr=-relax < %s \
+# RUN:     | FileCheck --check-prefix=CHECK-ASM %s
+# RUN: llvm-mc --triple loongarch64 --filetype=obj < %s \
```
Comment (SixWeining) [inline llvm/test/MC/LoongArch/Directives/option-pushpop.s:3]: --mattr=-relax

**Review by heiher — APPROVED** [review]
LGTM, Thanks.

**Review by SixWeining — APPROVED** [review]

**Pull request merged**